app/src/validators/object.js

```diff
@@ -120,6 +120,7 @@ const schema = {
   searchObjects: {
     headers: type.metadata(0),
     query: Joi.object({
+      userId: scheme.guid,
```

Comment:
For now, if this is a singular GUID, IIRC we should be using the uuidv4 scheme instead of scheme.guid, which permits one, or an array of guids.